prisma-fmt-wasm/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@prisma/prisma-fmt-wasm",
+  "version": null,
+  "description": "",
+  "main": "src/prisma_fmt_build.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/prisma/prisma-engines"
+  },
+  "author": "Prisma",
+  "license": "MIT",
```

Comment:
```suggestion
  "license": "Apache-2.0",
```

Reply:
To align with the prisma-engines license? I don't have any objections. It would be a change because we have been publishing prisma-fmt-wasm under MIT until now (the package.json is copied over from the prisma-fmt-wasm repo).

Reply:
Everything we publish is Apache 2.0, it would make sense to switch, I think.